privaterelay/views.py

```diff
@@ -164,6 +165,38 @@ def waitlist(request):
     return JsonResponse({'email': email, 'message': message}, status=status)
 
 
+# TODO: refactor to consolidate with emails.views
+@csrf_exempt
+def metrics_event(request):
+    request_data = json.loads(request.body)
+    # TODO: set and get ga_uuid in/from add-on, instead of looking it up
+    if (
+        not request.user.is_authenticated and
+        not request_data.get("api_token", False)
+       ):
+        raise PermissionDenied
+    api_token = request_data.get('api_token')
+    user_profile = _get_user_profile(request, api_token)
+    if not user_profile.ga_uuid:
```

Comment:
Do we want to move this `user_profile` check down to  L188 so that we can still send GA event data for new users who have never signed in? Thinking specifically about panel opens, "Join the Waitlist" and "Sign In" clicks, etc.